<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>271</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Page web</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>79</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>GET</string>
     </property>
     <widget class="QLineEdit" name="getValueLineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Valeur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="getNameLineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>169</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>POST</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Valeur1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="postName1LineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>as_Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="postValue1LineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>SmallFitz</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="postName2LineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>as_Pwd</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="postValue2LineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>KS7Y66W1Q</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Valeur2</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Télécharger</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileNameLineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>//home/smallfitz/Documents/MountyHall/0-Temp/result.html</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Url</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>360</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlLineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>http://games.mountyhall.com/mountyhall/MH_Play/PlayStart.php</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
